sql-plugin/src/main/scala/org/apache/spark/sql/rapids/arithmetic.scala

```diff
@@ -46,11 +46,15 @@ object GpuAnsi extends Arm {
 
   def assertMinValueOverflow(cv: GpuColumnVector, op: String): Unit = {
     withResource(minValueScalar(cv.dataType())) { minVal =>
-      withResource(cv.getBase.equalToNullAware(minVal)) { isMinVal =>
-        withResource(isMinVal.any()) { anyFound =>
-          if (anyFound.isValid && anyFound.getBoolean) {
-            throw new ArithmeticException(s"One or more rows overflow for $op operation.")
-          }
+      assertMinValueOverflowImp(minVal, cv, op)
+    }
+  }
+
+  def assertMinValueOverflowImp(minVal: Scalar, cv: GpuColumnVector, op: String): Unit = {
+    withResource(cv.getBase.equalToNullAware(minVal)) { isMinVal =>
+      withResource(isMinVal.any()) { anyFound =>
```

Comment:
There is already an util method to do this any check and it has some optimization.
[BoolUtils.isAnyValidTrue](https://github.com/NVIDIA/spark-rapids/blob/branch-22.04/sql-plugin/src/main/scala/com/nvidia/spark/rapids/BoolUtils.scala#L48)

Reply:
done